lib/ibm_watson/iam_token_manager.rb

```diff
@@ -73,6 +75,8 @@ def _token
     end
   end
 
+  private
+
   # Request an IAM token using an API key
   def _request_token
```

Comment:
Do we need to remove this `_` per https://github.ibm.com/Watson/developer-experience/issues/5311?

There are a few more - do we need to remove these as well?
![ruby_underscore](https://user-images.githubusercontent.com/4381558/45445822-e480eb00-b690-11e8-962e-2357d4872a9a.png)

Reply:
These methods are moved under `private` because they are only used by `IamTokenManager` or `WatsonService` class.

Reply:
so do we need both private and `_`? the comment we received was that methods should not be prefixed with `_`.

Reply:
yeah, it also said `The underscores should be removed or the methods should be marked as private` so I am removing `_` from methods that are not `private` only.

Reply:
ah sorry, I must have written the description incorrectly. The original comment was

>1. Is there any reason to start method name in `_`? Those methods are not private (if so use `private`) and I don't see any reason to use that.

I should have added this to the issue. Let me check with Chulki for clarification

Reply:
sounds good, let me know when we have more clarification from him

Reply:
Please do not prefix any method with `_` in ruby :)

Reply:
Chuliki confirmed starting methods with `_` is not ruby convention and we should remove them and mark them private instead.